WinUIGallery/.editorconfig

```diff
@@ -90,6 +90,9 @@ csharp_style_pattern_matching_over_is_with_cast_check = true:suggestion
 # IDE0016: Null check can be simplified
 csharp_style_throw_expression = true:suggestion
 
+# IDE0161: Convert to file-scoped namespace
+csharp_style_namespace_declarations = file_scoped:warning
```

Comment:
Not sure if this needs to be a warning? I don't see any danger in one or the other form.

Reply:
I'm following the way taken by CommunityToolkit Gallery and AI Dev Gallery. It's better this way to keep consistency.